server/util.js

```diff
@@ -16,9 +16,15 @@ if (config.http_proxy) {
 }
 
 const { performance } = require('node:perf_hooks')
-const get = (url, options, ext = false) => {
+const get = (url, options = {}, ext = false) => {
   const thisWreck = (ext && wreckExt) ? wreckExt : wreck
   const startTick = performance.now()
+  // Header with UA added for AWS WAF as it is required, otherwise it will block the request
+  options.headers = {
+    ...options.headers,
+    'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/58.0.3029.110 Safari/537.3'
```

Comment:
This should be a constant declared at the top of the file. And it should be something like 
`hapi-wreck/18 (cyltfr-app)` so we're not pretending to be something we're not.